src/core/energy_storage/electric_storage.jl

```diff
@@ -282,6 +294,13 @@ struct ElectricStorage <: AbstractElectricStorage
             @warn "Battery replacement costs (per_kwh) will not be considered because battery_replacement_year is greater than or equal to analysis_years."
         end
 
+        # if s.off_grid_flag && (can_net_meter || can_wholesale || can_export_beyond_nem_limit)
```

Comment:
why commented out?

Reply:
I updated in https://github.com/NREL/REopt.jl/pull/471/commits/875cbd25cad3acf194222234fd02a7462923fa0c